backend/server/schema.py

```diff
@@ -164,14 +211,20 @@ def resolve_predictions_by_round(root, _info) -> List[RoundPrediction]:
             for value in df.index.levels[0]
         ]
 
-        return (
+        round_predictions = (
             pd.DataFrame(list(query_set))
             .assign(cumulative_correct_count=calculate_cumulative_correct)
             .groupby(["match__round_number", "ml_model__name"])
             .mean()
-            .pipe(round_predictions)
         )
 
+        if round_number is not None:
+            round_predictions = round_predictions.query(
+                "match__round_number == @round_number"
```

Comment:
i think I haven't seen this syntax in your past PR's `@round_number`.. is that like the instance vars in ruby?

Reply:
Special syntax for the `query` and `eval` methods on `DataFrame` objects. Think of it as a form of string interpolation. In this method, all of the following are the same:
`"match__round_number == @round_number"`
`f"match__round_number == {round_number}"`
`"match__round_number == " + str(round_number)`

Reply:
[https://media.giphy.com/media/I5FGunA5wNSs8/giphy-downsized.gif](url)